README.md

```diff
@@ -1 +1,298 @@
+  </ul>
+</details>
+
+<details>
+<summary>Images</summary>
+<li>not yet available</li>
+
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **Add_house**
+- **Delete_house**
+- **Fetch/get_house**
+- **Authenticate_user**
+- **Reserve_house**
+
+- **[See overview of m]**
+  - The Real Estate Appointment Booking Application is a web-based software that aims to make the process of buying or selling a property more convenient and streamlined.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```sh
+ gem install rails
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd folder
+  git clone 
+  cd Rental-House-API
+  Open in your favorite editor
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd https://github.com/zakiyousufi/Rental-House-API.git
+  gem install
+  bundle install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails server or rails s
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  bin/rspec spec
+```
+
+### Deployment
+
+You can deploy this project using:
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+👤 **Ahmad Zaki Yousufi**
+
+- GitHub: [@zskiyousufi](https://github.com/zakiyousufi)
+- LinkedIn: [zakiyousufi](https://linkedin.com/in/zakiyousufi)
+
+
+👤 **Husseini Mudi**
+
+- GitHub: [@githubhandle](https://github.com/Profsain)
+- Twitter: [@twitterhandle](https://twitter.com/profsain)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/profsain)
+
+
+👤 **usirusen ikon**
+
+- GitHub: [@githubhandle](https://github.com/UsirusenIkon)
+- Twitter: [@twitterhandle](https://twitter.com/Usirusen_Ikon)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/usirusen-ikon)
+
+
+👤 **Soulemanou kunchiefuh babanou**
+
+- GitHub: [@githubhandle](https://github.com/soulemanou-software)
+- Twitter: [@twitterhandle](https://twitter.com/Mr_Babanou_237)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/soulemanou-kunchiefuh-babanou)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[List Transactions]**
+- [ ] **[Transaction Details]**
+
```

Comment:
The future features are not related to our project
Kindly add two or three future features related to the same project
They can be like `User profile page`
`User will able to add just a room`